services/drupal/web/modules/custom/epa_workflow/epa_workflow.module

```diff
@@ -950,7 +950,15 @@ function epa_workflow_theme($existing, $type, $theme, $path) {
         'flag_users' => [],
 
       ]
-    ]
+      ],
+      'epa_node_details' => [
+        'variables' => [
+          'nid' => NULL,
+          'rid' => NULL,
+          'revision_author' => NULL,
```

Comment:
Please rename this variable. This is the Editor in Chief user not the revision author.